src/components/ShowHide/__snapshots__/ShowHide.test.tsx.snap

```diff
@@ -6,6 +6,8 @@ exports[`<ShowHide /> Default story renders snapshot 1`] = `
 >
   <div
     class="show-hide"
+    role="button"
+    tabindex="0"
   >
     <button
```

Comment:
Looks like you have a button in a button here. Testing out the component in storybook, it looks like the escape key logic isn't working, and I feel like it's generally not that useful even if it did work. Popovers, tooltips, and modals need escape key logic, but I don't think I've ever seen it in a show/hide component like this one. I would just remove the `onKeyDown` logic entirely for this component. (All the code you're interacting with in this PR wasn't written by us but inherited from a collection of template components brought by the Big Medium group, so we haven't really looked at this stuff before to determine if we want/need it.)

I'll need some time to look over and think about the other components.

Reply:
Agreed that the button within a button is no bueno.

I haven't looked at the UI or analyzed this too deeply, but if removing `onKeyDown` makes sense, I say do it 👍.

Reply:
moved onKeyDown attribute to triggerElement